frontend/ui/data/koptoptions.lua

```diff
@@ -349,16 +349,25 @@ left to right or reverse, top to bottom or reverse.]]),
             {
                 name = "page_gap_height",
                 name_text = _("Page Gap"),
-                toggle = {C_("Page gap", "none"), C_("Page gap", "small"), C_("Page gap", "medium"), C_("Page gap", "large")},
-                values = {0, 8, 16, 32},
+                buttonprogress = true,
+                values = {0, 2, 4, 8, 16, 32, 64},
+                default_pos = 4,
                 default_value = 8,
-                args = {0, 8, 16, 32},
                 event = "PageGapUpdate",
+                args = {0, 2, 4, 8, 16, 32, 64},
                 enabled_func = function (configurable)
                     return optionsutil.enableIfEquals(configurable, "page_scroll", 1)
                 end,
                 name_text_hold_callback = optionsutil.showValues,
+                name_text_true_values = true,
+                name_text_unit = "px", -- values go through scaleBySize in the event handler
                 help_text = _([[In continuous view mode, sets the thickness of the separator between document pages.]]),
+                more_options = true,
+                more_options_param = {
+                    value_step = 1, value_hold_step = 10,
+                    value_min = 0, value_max = 256,
+                    precision = "%.1f",
```

Comment:
Decimal point?

Reply:
Yeah, necessary if you want to be able to icnrese in precise 'absolute) 1px steps on high-dpi devices and/or on the bigger end of the scale.

e.g., on a Clara 2E, 255 is something like 456px, and 256 458px. If you really really wanted 457px, being able to enter 455.5 helps doing just that ;).

Since it should be pretty niche, I left the step in integers, as that's what most people should be using ;).

TL;DR: `scaleBySize` handles floats just fine, so it can be useful ;).